Echo/PolySyncEchoHelp.hpp

```diff
@@ -0,0 +1,59 @@
+/**
+ * @file PolySyncEchoHelp.hpp
+ * @brief Help Class to augment Command Line Utility.
+ *
+ * PUBLIC_HEADER
+ */
+
+#ifndef POLYSYNCECHOHELP_HPP
+#define POLYSYNCECHOHELP_HPP
+
+#include <PolySyncDataModel.hpp>
+
+/**
+ * @namespace polysync
+ */
+namespace polysync
+{
+
+/**
+ * @brief The PolySyncEchoHelp class is a user interaction class to assist
+ *        end users with command line arguments/options for Echo utility.
+ *
+ */
+class PolySyncEchoHelp
```

Comment:
If this object is already in the `polysync` namespace, no need to prefix it with `PolySync`.

Use-case == `polysync::EchoHelp` vs. `polysync::PolySyncEchoHelp`